README.md

```diff
@@ -83,10 +83,10 @@ You will first need to create a Cornell lab account to [ask for access to and do
 - Run `sbatch filter2.awk raw_ebd_data_file.txt `script to group observations by locality. The script on the repository was the one used to create the USA dataset, where we kept checklists of June, July, December and January to prepare the USA-summer and USA-winter dataset. For Kenya no month filter was applied.
 This will create one file per hotspots, the files will be organized based on the hotspot name in `split-XXX/`  folders where each folder should only contain .csv files for hotspots `L*XXX`. 
 - Run  `data_processing/ebird/find_checklists.py` to filter the checklists you want to keep based on the ebd sampling event data. Typically this will only keep complete checklists and will create (for the USA dataset) 2 .csv files with the hotspots to keep corresponding to observations made in the summer and winter. 
-- to get the targets, run `data_processing/ebird/get_winter_targets.py` . Change the paths to the one where you created the `split-XXX/` folders.
+- to get the targets, run `data_processing/ebird/get_targets.py` . Change the paths to the one where you created the `split-XXX/` folders.
```

Comment:
Is this true for USA and Kenya?

Reply:
yes, the get_winter_targets file was renamed it seems.